context/app/static/js/components/Detail/Protocol/Protocol.jsx

```diff
@@ -26,12 +27,19 @@ function ProtocolLink(props) {
 function Protocol(props) {
   const { protocol_url } = props;
 
+  const lastWordPattern = new RegExp('([a-zA-Z0-9]+)$');
+  const matchedDoi = protocol_url.match(lastWordPattern)[0];
+
+  const protocolData = useProtocolData(matchedDoi, 1);
+
+  const doi = Object.keys(protocolData).length > 0 ? protocolData.protocol.doi : '';
+
   return (
     <SectionContainer id="protocols">
       <SectionHeader>Protocols</SectionHeader>
       <Divider />
       <StyledPaper>
-        <ProtocolLink protocolUrl={protocol_url} />
+        <ProtocolLink protocolUrl={doi} />
```

Comment:
DOI != URL

Reply:
What protocols.io returns as 'doi' is the href we're using, but I can see how that's confusing.

Reply:
Changed.